requirements.txt

```
@@ -1,33 +1,33 @@
-asyncio-throttle~=1.0
-aiohttp~=3.8.1
-aiodns~=3.0.0
-aiofiles==22.1.0
-aioredis~=2.0.1
-backoff~=2.1.2
-click<9.0
-cryptography~=38.0.1
+asyncio-throttle>=1.0,<=1.0.2
+aiohttp>=3.8.1,<=3.8.3
+aiodns>=3.0,<=3.0.0
+aiofiles>=22.1,<=22.1.0
+aioredis>=2.0.1<=2.0.1
+backoff>=2.1.2,<=2.1.2
+click<=8.1.3  # We don't know the lower bound
+cryptography>=38.0.1<=38.0.1
 Django~=3.2.15  # LTS version, switch only if we have a compelling reason to
-django-currentuser~=0.5.3
-django-filter~=22.1
-django-guid~=3.3
-django-import-export~=2.9.0
-django-lifecycle~=1.0.0
-djangorestframework~=3.13.1
-djangorestframework-queryfields~=1.0.0
-drf-access-policy~=1.1.2
-drf-nested-routers==0.93.4
+django-currentuser>=0.5.3,<0.5.3
```

```suggestion
django-currentuser>=0.5.3,<0.5.2
```
That was really unsatisfyable.

---

you meant `django-currentuser>=0.5.3,<=0.5.3`

---

Not really sure which format i prefer. They may be slightly different, but not so much that i really care. But most important: Are we sure that dependabot will never touch the lower bound and only ever try to push out the upper one?

---

I've been reading the docs for dependabot and that's my understanding. I'll be watching the dependabot PRs in the future for exactly this also.